File: 3191-minimum-operations-to-make-binary-array-elements-equal-to-one-i/3191-minimum-operations-to-make-binary-array-elements-equal-to-one-i.java
class Solution {
    public int minOperations(int[] nums) {
        int op = 0;
        int n = nums.length;
        for(int i=0;i<n;i++){
            if(nums[i] == 0){
                if(i+2 < n){
                    nums[i] ^=1;
                    nums[i+1] ^=1;
                    nums[i+2] ^=1;
                    op++;

                }
                else{
                    return -1;
                }
            }
        }
        return op;

        
    }
}

File: 3191-minimum-operations-to-make-binary-array-elements-equal-to-one-i/3191-minimum-operations-to-make-binary-array-elements-equal-to-one-i.py
class Solution(object):
    def minOperations(self, nums):
        op = 0
        n = len(nums)
        for i in range(n):
            if nums[i] == 0:
                if i+2 < n:
                    nums[i] ^= 1
                    nums[i+1] ^= 1
                    nums[i+2] ^= 1
                    op+=1
                else:
                    return -1
        return op
        """
        :type nums: List[int]
        :rtype: int
        """
        